odoo/addons/base/tests/common.py

```diff
@@ -29,10 +29,65 @@ def setUpClass(cls):
         # Hack to use with_context and avoid manual context dict modification
         cls.env = cls.env['base'].with_context(**DISABLED_MAIL_CONTEXT).env
```

Comment:
Note that this hides a lot of stuff, unsure it is a good idea (might be out of scope for this task, just pointing it out)

Reply:
Yup, I had to remove this context for some tests, but it's something @Feyensv proposed (in terms of perfs I think?)